tbx/project_styleguide/templatetags/navigation_tags.py

```diff
@@ -2,4 +2,5 @@
 from tbx.navigation.templatetags.navigation_tags import register
 
 override_tag(register, name="primarynav", default_html="")
+override_tag(register, name="primarynavmobile", default_html="")
```

Comment:
I'd missed this previously and the pattern library was showing the navigation based on wagtail's site settings, not based on the yaml.